lib/plugins/ticketService.js

```diff
@@ -25,6 +25,7 @@ const mockProof = {
     'P8T2CezuFY#z279yHL6HsxRzCPU78DAWgZVieb8xPK1mJKJBbP8T2CezuFY',
   capability: 'did:v1:uuid:c37e914a-1e2a-4d59-9668-ee93458fd19a',
   capabilityAction: 'write',
+  invocationTarget: 'urn:zcap:root:ledger',
```

Comment:
Please fix this too.

Reply:
replaced it with a mock did here: https://github.com/veres-one/veres-one/pull/44/commits/4d17de37351a3f067d74dc016337b420f9563659